### PR TITLE
About section for KF 1.4 docs

### PR DESCRIPTION
**Pull request by judyheflin**
The following pages are staged for the About section in the KF 1.4 Hugo docs site:
- About 
- AWS Features for Kubeflow
- Releases and Versioning
- Amazon EKS and Kubeflow Compatibility
- Security

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by ryansteakley** [comment]
Should we include more information about the benefits of CloudWatch in the compute section rather than have a one-liner and have a complete section for it?


Can include info such as 

> CloudWatch Logs and Metrics

> * Integrate Kubeflow with [AWS ContainerInsights](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/ContainerInsights.html) (https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/ContainerInsights.html) to  collect, aggregate, and summarize metrics and logs from your containerized applications and microservices. 
> * Kubernetes by default has a upper limit on the log size, you lose any logs if it crosses that limit. Pod logs will persist in Cloudwatch
> * If the pod gets evicted or rescheduled etc. user losses the logs, with CW integration you persist the logs so it helps with troubleshooting

**Comment by judyheflin** [comment]
> 

Changing this to: 
## CloudWatch Logs and Metrics
* Integrate Kubeflow on AWS with **[Amazon CloudWatch](https://aws.amazon.com/cloudwatch/)** for persistent logging and metrics on EKS clusters and Kubeflow pods.
* Use [AWS ContainerInsights](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/ContainerInsights.html) to collect, aggregate, and summarize metrics and logs from your containerized applications and microservices.
I'll add some of the more specific features on the CloudWatch page directly. Thanks!

File: docs/about/releases.md [hunk 1]
```diff
@@ -1,26 +1,23 @@
...
-Starting with Kubeflow v1.4, the development of the AWS distribution of Kubeflow is done on the [`main`](https://github.com/awslabs/kubeflow-manifests/tree/main) branch. The `main` branch contains only the delta from the released manifests in the `kubeflow/manifests` repository and additional components required for the AWS distribution.
-
-### Versioning
++++
+title = "Releases and Versioning"
+description = "Familiarize yourself with Kubeflow on AWS release cadences and version naming conventions"
+weight = 20
++++
 
 Kubeflow on AWS releases are built on top of Kubeflow releases and therefore use the following naming convention: `{KUBEFLOW_RELEASE_VERSION}-aws-b{BUILD_NUMBER}`.
```
Comment (goswamig) [inline docs/about/releases.md:7]: on top of open source Kubeflow releases

File: docs/about/security.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+kustomize build distributions/aws/aws-telemetry | kubectl apply -f -
+```
+
+### Deactivate usage tracking
+
+**Before deploying Kubeflow:** 
+
+You can deactivate usage tracking by skipping the telemetry component installation in one of two ways:
+
+1. For single line installation, comment out the `aws-telemetry` line in the `kustomization.yaml` file:
```
Comment (surajkota) [inline docs/about/security.md:38]: I think we can add one example line to a kustomization file

**Review by goswamig — APPROVED** [review]
LGTM, minor change

File: docs/about/security.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    ```
+    # ./../aws-telemetry
+    ```
+1. For individual component installation, **do not** install the `aws-telemetry` component: 
+    ```
+    # AWS Telemetry - This is an optional component. See usage tracking documentation for more information
+    kustomize build distributions/aws/aws-telemetry | kubectl apply -f -
```
Comment (surajkota) [inline docs/about/security.md:45]: Update this command:
`kustomize build awsconfigs/common/aws-telemetry | kubectl apply -f -`

File: docs/about/security.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+* [Security best practices for Amazon Elastic Kubernetes Service (EKS)](https://aws.github.io/aws-eks-best-practices/security/docs/)  
+* [Security best practices for AWS Secrets Manager](https://docs.aws.amazon.com/secretsmanager/latest/userguide/best-practices.html)  
+* [Security best practices for Amazon Relational Database Service (RDS)](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_BestPractices.Security.html)  
+* [Security best practices for Amazon Simple Storage Service (S3)](https://docs.aws.amazon.com/AmazonS3/latest/userguide/security-best-practices.html)  
+* [Security in Amazon Route53](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/security.html)  
+* [Security in Amazon Certificate Manager (ACM)](https://docs.aws.amazon.com/acm/latest/userguide/security.html)  
+* [Security best practices for Amazon Cognito user pools](https://docs.aws.amazon.com/AmazonS3/latest/userguide/security-best-practices.html)  
+* [Security in Amazon Elastic Load Balancing (ELB)](https://docs.aws.amazon.com/elasticloadbalancing/latest/userguide/security.html)
+
+## Usage tracking
```
Comment (surajkota) [inline docs/about/security.md:18]: It will be good to put this as a separate page under about

**Review by ryansteakley — APPROVED** [review]

**Review by akartsky — APPROVED** [review]

File: docs/about/usage-tracking.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+AWS uses customer feedback and usage information to improve the quality of the services and software we offer to customers. We have added usage data collection to the AWS Kubeflow distribution in order to better understand customer usage and guide future improvements. Usage tracking for Kubeflow is activated by default, but is entirely voluntary and can be deactivated at any time. 
+
+Usage tracking for Kubeflow on AWS collects the instance ID used by one of the worker nodes in a customer’s cluster. This data is sent back to AWS once per day. Usage tracking only collects the EC2 instance ID where Kubeflow is running and does not collect or export any other data to AWS. If you wish to deactivate this tracking, instructions are below. 
+
+### Activate usage tracking
+
+Usage tracking is activated by default. If you deactivated usage tracking for your Kubeflow deployment and would like to activate it after the fact, you can do so at any time with the following command:
+
+```
+kustomize build distributions/aws/aws-telemetry | kubectl apply -f -
```
Comment (akartsky) [inline docs/about/usage-tracking.md:16]: as pointed out by Suraj 

Update this line to 
`kustomize build awsconfigs/common/aws-telemetry | kubectl apply -f -`

**Review by akartsky — CHANGES_REQUESTED** [review]

File: docs/about/usage-tracking.md (hunk 5 above)
Comment (judyheflin) [reply docs/about/usage-tracking.md:16]: Did that in my latest push: https://github.com/awslabs/kubeflow-manifests/pull/188/commits/a15d5bd5fb7ed95138c5997602aa6b5f5bfbff90#diff-a693f8b2d8b72223fa6a82d2d8273df89eb8db076fe2865fbae51d14fcb4e3df

**Review by surajkota — APPROVED** [review]

**Review by akartsky — APPROVED** [review]

**Pull request merged**